lib/Jobs/jobExecutionPropTypes.js

```diff
@@ -18,10 +18,10 @@ const jobExecutionPropTypes = PropTypes.shape({
   }).isRequired,
   progress: PropTypes.shape({
     jobExecutionId: PropTypes.string,
-    current: PropTypes.number.isRequired,
+    current: PropTypes.number,
     total: PropTypes.number.isRequired,
   }).isRequired,
-  startedDate: PropTypes.string.isRequired,
+  startedDate: PropTypes.string,
   completedDate: PropTypes.string,
   status: PropTypes.string.isRequired,
   uiStatus: PropTypes.string.isRequired,
```

Comment:
Could you please make it optional, as for data-export there won't be such a field. [Here](https://github.com/folio-org/mod-data-export/blob/69df0703653e2d07a25759c6d66b4ee0efc03be5/ramls/samples/jobExecution.sample) is the list of available fields.